Makefile

```diff
@@ -34,7 +34,10 @@ clean:
 build:
 	$(GO) build $(BUILDFLAGS) ./test/...
 
-.PHONY: clean test build fmt
+build-all:
+	$(GO) test -run=nope -failfast -short ./test/...
```

Comment:
what does that do?

Reply:
Just a way to compile everything, including the tests, locally - there’s an equivalent target in jx and I just wanted the same thing here.